indigo_app/views/works.py

```diff
@@ -72,8 +72,32 @@ def get_context_data(self, **kwargs):
         works = WorkViewSet.queryset.filter(country=self.country, locality=self.locality)
         context['works_json'] = json.dumps(serializer.to_representation(works))
 
+        return context
+
+
+class PlaceTasksView(AbstractAuthedIndigoView, PlaceBasedView, TemplateView, MultipleObjectMixin):
```

Comment:
I'm being silly. We can just use [ListView](http://ccbv.co.uk/projects/Django/1.11/django.views.generic.list/ListView/) here -- it pulls in MultipleObjectMixin automatically. So `AbstractAuthedIndigoView, PlaceBasedView, ListView`, then you can define `get_queryset()` to return `Task.objects.filter(...)` and everything should be injected for you!